dissect/target/plugin.py

```diff
@@ -889,6 +895,7 @@ def find_plugin_functions(target: Target, patterns: str, compatibility: bool = F
             pattern += "*"
 
         if wildcard or treematch:
+            matches = 0
```

Comment:
Why not use a boolean for this?

```suggestion
            matches = False
```